projects/net-snmp/build.sh

```diff
@@ -16,16 +16,25 @@
 ################################################################################
 
 # build project
-./configure --with-openssl=/usr --with-defaults --with-logfile="/dev/null" --with-persistent-directory="/dev/null"
-# net-snmp build is not parallel-make safe; do not add -j
-make
+#./configure --with-openssl=/usr --with-defaults --with-logfile="/dev/null" --with-persistent-directory="/dev/null"
+#make
```

Comment:
These two lines should not be commented out

Reply:
This is a local change that got pushed out accidentally.